languageserver/src/client.ts

```diff
@@ -1,8 +1,27 @@
 import {Octokit} from "@octokit/rest";
+import { Agent } from "node:https";
+import { readFileSync } from "fs";
 
 export function getClient(token: string, userAgent?: string): Octokit {
-  return new Octokit({
-    auth: token,
-    userAgent: userAgent || `GitHub Actions Language Server`
-  });
+  const selfSignedCertPath = process.env.PATH_TO_SELF_SIGNED_CERT;
```

Comment:
This env var should be set by the user. I'm not sure what the best way to set this is. It looks like you can set env vars for the extension in `launch.json` ([reference](https://stackoverflow.com/questions/62493776/how-should-the-environment-variables-of-the-vscode-plugin-be-set))

Reply:
Locally I can populate this env var by exporting a variable in zsh. Not sure if this will work on the installed extension.

Reply:
@jtamsut I believe `launch.json` would be for setting environment variables *when developing* the extension.

I don't think that's doable for installed extensions. I think we should lean on the existing mechanisms for VS Code Extension settings here.